addons/account/static/src/css/account.css

```diff
@@ -28,13 +28,18 @@
     clear: both;
     float: right;
     min-width: 260px;
-    padding-top: 20px;
+    background-color: #FAFAFB;
+    margin-left: auto;
 }
 
 .oe_account_terms {
     flex: auto !important;
 }
 
+.o_form_view .oe_invoice_lines_tab .oe_invoice_outstanding_credits_debits {
+        width: 90%;
+    }
```

Comment:
indentation problem here and not really of a fan of using a width 90% like this, why do you need this ?